docs/content/documentation/templates/overview.md

```diff
@@ -64,7 +64,8 @@ Zola adds a few filters in addition to [those](https://tera.netlify.com/docs/#fi
 in Tera.
 
 ### markdown
-Converts the given variable to HTML using Markdown. Shortcodes won't work within this filter.
+Converts the given variable to HTML using Markdown. Please note that shortcodes evaluated by this filter won't be able to access any context variables.
```

Comment:
I believe config context (and maybe some other things) will be available inside the markdown. Maybe something like:
```
Please note that shortcodes evaluated by this filter cannot currently access the current rendering context. `config` will be available, but accessing `section` or `page` (among others) from a shortcode called within the `markdown` filter will prevent your site from building. See [this discussion](https://github.com/getzola/zola/pull/1358).
```

Reply:
Done!

(Minus one word; "currently access the current" read a bit awkward, so I dropped "currently".)